dev/SapphireTestReporter.php

```diff
@@ -1,6 +1,10 @@
 <?php
-if(!class_exists('PHPUnit_Framework_TestResult', false)) require_once 'PHPUnit/Framework/TestResult.php';
-if(!class_exists('PHPUnit_Framework_TestListener', false)) require_once 'PHPUnit/Framework/TestListener.php';
+if(!class_exists('PHPUnit_Framework_TestResult', false)) {
```

Comment:
see https://github.com/silverstripe/silverstripe-framework/pull/3958 as a half-way-house (allowing the autoloader to find the class)

Personally, I'd rather just see it gone